marvin.py

```diff
@@ -532,6 +558,21 @@ def delrule(self, update):
 
             return
 
+    def pin_if_necessary(self, to_pin, submission):
+        """ (Telegram command)
+        Pin reddit post if necessary
+        :param to_pin: the message to pin
+        :param submission: the reddit post
+        """
+        for autopin_rule in self.auto_pinned_posts:
+            if str(submission.title.lower()).find(autopin_rule["text"]) != -1:
+                for authors_pin in autopin_rule["user"]:
+                    if (authors_pin == submission.author.name.lower()):
+                        self.updater.bot.pin_chat_message(to_pin.chat_id, to_pin.message_id, disable_notification=True)
+                        return
+        return
```

Comment:
Questo return qui è abbastanza inutile mi sa

Reply:
no, se ha trovato una regola per cui pinnare, non ha senso controllare le altre

Reply:
Nel for certo, è il return fuori dal for che non serve a nulla

Reply:
ah ok, lo tolgo